Doc/library/subprocess.rst

```diff
@@ -693,9 +693,12 @@ quoted appropriately to avoid
 `shell injection <https://en.wikipedia.org/wiki/Shell_injection#Shell_injection>`_
 vulnerabilities.
 
-When using ``shell=True``, the :func:`shlex.quote` function can be
-used to properly escape whitespace and shell metacharacters in strings
-that are going to be used to construct shell commands.
+When using ``shell=True`` on **Unix** systems, the :func:`shlex.quote`
```

Comment:
It looks like this may still be a little too broad; it gives the impression that `shlex.quote` makes things safe on *all* Unix platforms.  Maybe this whole paragraph should go away and instead an extra line be added to the previous one, along the lines of "In some situations, `shlex.quote` can be used to do this escaping."   We could make "some situations" a link to the warnings in the `shlex.quote` docs about where it's useful..

Reply:
Good point, it does have duplicated information from the shlex warning and is a bit too broad. I've linked it off to the warning in `shlex` and merged it in with the previous paragraph.